### DISCUSSION TITLE
Intrusive transportation case, when station is common between supply and consumer

### DISCUSSION THREAD

**Discussion by kaomoneus** [voxeltycoon/issues#739] [General]
**Describe the bug**
There is a case when factory and consumer (business) are located nearby. In this case use of cars looks intrusive. Perhaps this is a bug in gameplay logic. Just look at this screenshot:
![изображение](https://user-images.githubusercontent.com/857042/117194390-196b7e80-adf5-11eb-9f64-72df3b76fdef.png)
* There are 3 warehouses:
   * Coal
   * Iron ore
   * Iron bars

* There is a station which accepts all of them
   * Also, consumer (Industrial building II) is feasible from that station. It is also indicated by logistic links.
 
But this way doesn't work. Consumer can't take iron bars neither from station nor from warehouse. And it looks quite weird.

**Steps to reproduce**
See screen shot.

**Build version**
0.85.1

**Platform**
MBP Late 2013 

**Reproducibility**
Every time

**Logs/saves**
Let me know if you need them

**Comment by DBT85** [comment]
Goods have to be delivered to a station by train or truck in order to be automagically teleported to a shop. This is intended. The logistic link is there to show that, if delivered by train or truck, the goods will teleport to the shop. I can understand why it might seem odd given that you have a WH full of bars connected to the same station. 

My preference is to play the game as I feel it "should" be, in that all goods need to be delivered by truck to the shop, for this I used Refreshfr's small truck station mod.

You have 2 options. Cheese the system by having a truck sit at the station and load/unload the iron bars which will then teleport them to the shop, or set up a new station near the shop to deliver them properly.

**Comment by greeny** [comment]
hello, you have selected the truck station's logistic tab, so it shows where can trucks from that station load or unload. It doesn't actually connect two buildings that are shown there. This is the intended way it works. If you want to sell a resource, you need to deliver it with a truck.

**Comment by kaomoneus** [comment]
> If you want to sell a resource, you need to deliver it with a truck.

Exactly. But in this case it's counterintuitive. I believe, that you guys (and me) know it only because of ttd experience, but this is not how it should be.

Imagine absolutely new user joins the game. How she should guess it? What her reaction would be once she'll figure out the solution?

**Comment by greeny** [comment]
>  I believe, that you guys (and me) know it only because of ttd experience, but this is not how it should be.

I know it because it was in the tutorial.

**Comment by andrewpey** [comment]
Looks like a duplicate of https://github.com/voxeltycoon/issues/discussions/475.
